[libtheora] build failure

### Operating system

Windows 10

### Compiler

Cmake

### Steps to reproduce the behavior

```Shell
C:\Decomps\CD\vcpkg>vcpkg.exe install glew sdl2 libogg libtheora libvorbis --triplet=x64-windows-static
Computing installation plan...
The following packages are already installed:
    glew:x64-windows-static@2.2.0#6
    libogg:x64-windows-static@1.3.5#2
The following packages will be built and installed:
    libtheora:x64-windows-static@1.2.0alpha1-20170719#7
    libvorbis:x64-windows-static@1.3.7#3
    sdl2:x64-windows-static@2.32.2
Detecting compiler hash for triplet x64-windows-static...
Compiler found: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.43.34808/bin/Hostx64/x64/cl.exe
Restored 0 package(s) from C:\Users\######\AppData\Local\vcpkg\archives in 408 us. Use --debug to see more details.
glew:x64-windows-static is already installed
libogg:x64-windows-static is already installed
Installing 1/3 libtheora:x64-windows-static@1.2.0alpha1-20170719#7...
Building libtheora:x64-windows-static@1.2.0alpha1-20170719#7...
-- Using cached xiph-theora-fa5707d68c2a4338d58aa8b6afc95539ba89fecb.tar.gz
-- Cleaning sources at C:/Decomps/CD/vcpkg/buildtrees/libtheora/src/39ba89fecb-09b81b9348.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Decomps/CD/vcpkg/downloads/xiph-theora-fa5707d68c2a4338d58aa8b6afc95539ba89fecb.tar.gz
-- Applying patch 0001-fix-uwp.patch
-- Using source at C:/Decomps/CD/vcpkg/buildtrees/libtheora/src/39ba89fecb-09b81b9348.clean
-- Found external ninja('1.12.1').
-- Configuring x64-windows-static
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: "C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v
    Working Directory: C:/Decomps/CD/vcpkg/buildtrees/libtheora/x64-windows-static-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\Decomps\CD\vcpkg\buildtrees\libtheora\config-x64-windows-static-dbg-CMakeCache.txt.log
      C:\Decomps\CD\vcpkg\buildtrees\libtheora\config-x64-windows-static-rel-CMakeCache.txt.log
      C:\Decomps\CD\vcpkg\buildtrees\libtheora\config-x64-windows-static-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:269 (vcpkg_execute_required_process)
  ports/libtheora/portfile.cmake:21 (vcpkg_cmake_configure)
  scripts/ports.cmake:203 (include)

error: building libtheora:x64-windows-static failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle libtheora:x64-windows-static: 1.3 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+libtheora
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[libtheora]+Build+error+on+x64-windows-static&body=Copy%20issue%20body%20from%20C%3A%2FDecomps%2FCD%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md
```

### Failure logs

[1/2] "C:/Program Files/CMake/bin/cmake.exe" -E chdir "../../x64-windows-static-dbg" "C:/Program Files/CMake/bin/cmake.exe" "C:/Decomps/CD/vcpkg-master/buildtrees/libtheora/src/39ba89fecb-09b81b9348.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/Decomps/CD/vcpkg-master/packages/libtheora_x64-windows-static/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DVERSION:STRING=1.2.0alpha1-20170719" "-DUSE_X86=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Decomps/CD/vcpkg-master/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-static" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Decomps/CD/vcpkg-master/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=static" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Decomps/CD/vcpkg-master" "-D_VCPKG_INSTALLED_DIR=C:/Decomps/CD/vcpkg-master/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../../x64-windows-static-dbg/CMakeCache.txt 
"C:/Program Files/CMake/bin/cmake.exe" -E chdir "../../x64-windows-static-dbg" "C:/Program Files/CMake/bin/cmake.exe" "C:/Decomps/CD/vcpkg-master/buildtrees/libtheora/src/39ba89fecb-09b81b9348.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/Decomps/CD/vcpkg-master/packages/libtheora_x64-windows-static/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DVERSION:STRING=1.2.0alpha1-20170719" "-DUSE_X86=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Decomps/CD/vcpkg-master/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-static" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Decomps/CD/vcpkg-master/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=static" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Decomps/CD/vcpkg-master" "-D_VCPKG_INSTALLED_DIR=C:/Decomps/CD/vcpkg-master/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!
[2/2] "C:/Program Files/CMake/bin/cmake.exe" -E chdir ".." "C:/Program Files/CMake/bin/cmake.exe" "C:/Decomps/CD/vcpkg-master/buildtrees/libtheora/src/39ba89fecb-09b81b9348.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Decomps/CD/vcpkg-master/packages/libtheora_x64-windows-static" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DVERSION:STRING=1.2.0alpha1-20170719" "-DUSE_X86=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Decomps/CD/vcpkg-master/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-static" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Decomps/CD/vcpkg-master/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=static" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Decomps/CD/vcpkg-master" "-D_VCPKG_INSTALLED_DIR=C:/Decomps/CD/vcpkg-master/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../CMakeCache.txt 
"C:/Program Files/CMake/bin/cmake.exe" -E chdir ".." "C:/Program Files/CMake/bin/cmake.exe" "C:/Decomps/CD/vcpkg-master/buildtrees/libtheora/src/39ba89fecb-09b81b9348.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Decomps/CD/vcpkg-master/packages/libtheora_x64-windows-static" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DVERSION:STRING=1.2.0alpha1-20170719" "-DUSE_X86=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Decomps/CD/vcpkg-master/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-static" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Decomps/CD/vcpkg-master/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=static" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Decomps/CD/vcpkg-master" "-D_VCPKG_INSTALLED_DIR=C:/Decomps/CD/vcpkg-master/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!
ninja: build stopped: subcommand failed.


### Additional context

Hello! 

I was trying to build a Decomp of Sonic CD but kept getting this crash. Is there something I missed or did wrong?

Thank You! :3

> [config-x64-windows-static-out.log](https://github.com/user-attachments/files/19330414/config-x64-windows-static-out.log) this is for sonic mania

>> CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
>>  Compatibility with CMake < 3.5 has been removed from CMake.

Please try a lower version (like `3.30.1`) of CMake as workaround.

See https://github.com/microsoft/vcpkg/pull/44147

--- a/ports/libtheora/CMakeLists.txt
+++ b/ports/libtheora/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.30)
 project(theora LANGUAGES C)
 
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}")

--- a/ports/libtheora/vcpkg.json
+++ b/ports/libtheora/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libtheora",
   "version-string": "1.2.0alpha1-20170719",
-  "port-version": 7,
+  "port-version": 8,
   "description": "Theora is a free and open video compression format from the Xiph.org Foundation.",
   "homepage": "https://github.com/xiph/theora",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5282,7 +5282,7 @@
     },
     "libtheora": {
       "baseline": "1.2.0alpha1-20170719",
-      "port-version": 7
+      "port-version": 8
     },
     "libtins": {
       "baseline": "4.5",

--- a/versions/l-/libtheora.json
+++ b/versions/l-/libtheora.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "875c4b56b5fbda529ab8a2efb67209b35408134a",
+      "version-string": "1.2.0alpha1-20170719",
+      "port-version": 8
+    },
+    {
       "git-tree": "f6cc1235f841d9f9a7b7d1a278dc63b788c63659",
       "version-string": "1.2.0alpha1-20170719",
       "port-version": 7

